network/phonebook.go

```diff
@@ -47,50 +47,78 @@ type Phonebook interface {
 	// UpdateConnectionTime will update the provisional connection time.
 	// Returns true of the addr was in the phonebook
 	UpdateConnectionTime(addr string, provisionalTime time.Time) bool
+
+	// ReplacePeerList merges a set of addresses with that passed in for networkName
+	// new entries in dnsAddresses are being added
+	// existing items that aren't included in dnsAddresses are being removed
+	// matching entries don't change
+	ReplacePeerList(dnsAddresses []string, networkName string)
+
+	// ExtendPeerList adds unique addresses to this set of addresses
+	ExtendPeerList(more []string, networkName string)
 }
 
-type phonebookData struct {
```

Comment:
could you please add a comment explaining the phonebook internals? How about renaming **phonebookEntryData** to addressData since it holds info about addresses - timing and inclusion to networks.

Reply:
name changed to addressData. 
Added comment to addressData.